crates/services/txpool_v2/src/service.rs

```diff
@@ -432,7 +430,8 @@ where
             };
 
             if metrics {
-                record_tx_size(&checked_tx)
+                let size = checked_tx.metered_bytes_size();
+                txpool_metrics().tx_size.observe(size as f64);
             };
```

Comment:
Do we want to record the size of inserted transaction or size of all transactions that entered the pool? If the first one, then maybe it makes sense to move this metric into `TxPool::insert` function. If the second, then we need to do it along with `.number_of_transactions_pending_verification.inc()`.

Reply:
As discussed, we're interested in logging transactions that has been _inserted_ into the TxPool, meaning: every transaction that passed all validations and is actually being inserted.

Other cases (i.e. log rejected transactions) are still to be handled, as per the original issue: https://github.com/FuelLabs/fuel-core/issues/1937 - this will be delivered as a separate PR.